specification/communication/data-plane/Chat/stable/2021-09-07/communicationserviceschat.json

```diff
@@ -1688,6 +1700,38 @@
           "example": "Bob Admin"
         }
       }
+    },
+    "DeletionStrategy": {
+      "type": "object",
+      "discriminator": "strategy",
+      "properties": {
+        "strategy": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "strategy"
+      ]
+    },  
+    "DeleteBasedOnCreationStrategy": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/DeletionStrategy"
+        },
+        {
+          "type": "object",
+          "properties": {
+            "daysAfterCreation":{
+              "type": "integer",
+              "format": "int32",
+              "description": "Only 90 is accepted for now"
```

Comment:
In the description can you also include an explanation of what this setting does, not just the liminations?